sfa_dash/templates/dash/data.html

```diff
@@ -30,7 +30,12 @@
 
 {% if page_title is defined or breadcrumb is defined %}
 <h2 class="page-title">
-{{ page_title }}{{ breadcrumb|safe }}
+{# display a breadcrumb if present, otherwise use page_title #}
+{% if breadcrumb is defined %}
+{{ breadcrumb|safe }}
+{% else %}
+{{ page_title }}
+{% endif %}
```

Comment:
what does this address?

Reply:
`page_title` is used to set a static non-link heading on the page, and `breadcrumb` is expected to be the series of links separated with `/`. This is why the page title setting above first checks for a metadata object to pull a name from. The individual forecast/obs pages only have a breadcrumb.

Reply:
ah, so this doesn't actually change anything or fix an issue, it's just more explicit about what's going on?

Reply:
To get the page `<title>` attributes working I added `page_title` to pages that were missing it, so separating was to explicitly ensure we don't print both.

Reply:
To answer your original question, This is to allow for providing a `page_title` and a `breadcrumb` without printing both at the top of the page. In that case, the `page_title` is only used in the `<title>` tag and the `breadcrumb` is displayed at the top of the page. For example, on the /Sites where we want a `page_title` of "Sites" and a breadcrumb like `/<a href="/sites">Sites</a>`.